### PR TITLE
Compiles gor from source

### PR DESCRIPTION
**Pull request by ptrr**

### CODE REVIEW

File: recipes/default.rb [hunk 1]
```diff
@@ -2,24 +2,34 @@
 # Cookbook Name:: gor
 # Attributes:: default
 #
-# Author:: Wouter de Vos <wouter@springest.com>
+# Author:: Wouter de Vos <wouter@springest.com>, Peter de Ruijter <edge@springest.com>
```
Comment (dennispaagman) [inline recipes/default.rb:5]: :trollface:

Comment (ptrr) [inline recipes/default.rb:5]: :copyright:

File: recipes/default.rb [hunk 2]
```diff
@@ -2,24 +2,34 @@
...
+
+include_recipe "golang::default"
+
+
+git "~/gor" do
+  repository node['gor']['distribution']['repository']
+  reference node['gor']['distribution']['revision']
+  action :sync
+  notifies :run, 'execute[compile-gor-source]', :immediately
+  not_if do ::File.exists?("/usr/local/bin/gor") end
```
Comment (wearethefoos) [inline recipes/default.rb:18]: Code conventions:

``` ruby
not_if { ::File.exists?("/usr/local/bin/gor") }
```

File: recipes/default.rb [hunk 3]
```diff
@@ -2,24 +2,34 @@
...
-  source 'gor-linux-amd64'
-  owner 'root'
-  mode 0744
-  action :create
-  only_if( "uname -m | grep x86_64" )
+
+include_recipe "golang::default"
+
+
+git "~/gor" do
```
Comment (miry) [inline recipes/default.rb:13]: Why do you use home folder? I think `/opt/gor` or `/usr/local/src/gor` would be much better.

**Comment by lizgene** [comment]
Super late to the game, but closing this since its last comment was from over a year ago. I'm cleaning this up to mark it as unmaintained by our team, but let us know if you'd like to keep working on it and/or take the repository over! :smile:

**Comment by ptrr** [comment]
Wow, is this mine? Haha. Okay. :P

**Pull request closed without merging**